crates/networking/rpc/engine/payload.rs

```diff
@@ -32,9 +32,12 @@ impl RpcHandler for NewPayloadV3Request {
             return Err(RpcErr::BadParams("Expected 3 params".to_owned()));
         }
         Ok(NewPayloadV3Request {
-            payload: serde_json::from_value(params[0].clone())?,
-            expected_blob_versioned_hashes: serde_json::from_value(params[1].clone())?,
-            parent_beacon_block_root: serde_json::from_value(params[2].clone())?,
+            payload: serde_json::from_value(params[0].clone())
+                .map_err(|_| RpcErr::WrongParam("payload".to_string()))?,
```

Comment:
do you think it makes sense yo have both `BadParams` and `WrongParam`? out of scope of this pr though

Reply:
They have different error codes and hive expects different test cases to return different error codes so we have to comply with it